xml/System.Security.Cryptography.X509Certificates/X509Chain.xml

```diff
@@ -464,9 +464,31 @@
 
  A chain element consists of an <xref:System.Security.Cryptography.X509Certificates.X509Certificate2> object, an <xref:System.Security.Cryptography.X509Certificates.X509ChainStatus> structure, and an extra information string.
 
+ The `ChainElements` collection is ordered from the end-entity (leaf) certificate at index 0, through any intermediate certificates, to the trust anchor (root certificate) at the final index. This ordering is consistent across all platforms:
 
+ - On Windows, this matches the `CERT_CHAIN_CONTEXT` structure where `rgpChain[0]` is the end certificate and `rgpChain[cChain–1]` is the final chain element (root).
+ - On Linux and macOS, this matches OpenSSL's `X509_STORE_CTX_get0_chain()` which returns a `STACK_OF(X509)` ordered from leaf to root.
 
 ## Examples
+ The following code example demonstrates the ordering of chain elements:
+
+ ```csharp
```

Comment:
This example should go in a separate .cs file similar to the example on line 494.